Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebButtonsView.swift

```diff
@@ -27,15 +27,15 @@ struct PayPalWebButtonsView: View {
 
                 switch selectedFundingSource {
                 case .paypalCredit:
-                    PayPalCreditButton.Representable(color: .gold, size: .full) {
+                    PayPalCreditButton.Representable(color: .gold, size: .regular) {
                         payPalWebViewModel.paymentButtonTapped(funding: .paypalCredit)
                     }
                 case .paylater:
                     PayPalPayLaterButton.Representable(color: .silver, edges: .softEdges, size: .full) {
```

Comment:
Since `.full `no longer exists, should this button be sized as `.regular`?

```suggestion
                    PayPalPayLaterButton.Representable(color: .silver, edges: .softEdges, size: .regular) {
```